NineAnimator/Models/Anime Source/animeunity.it/AnimeUnity+Search.swift

```diff
@@ -57,31 +62,23 @@ extension NASourceAnimeUnity {
                         let data = responseContent
                         let utf8Text = String(data: data, encoding: .utf8) ?? String(decoding: data, as: UTF8.self)
                         let  bowl = try SwiftSoup.parse(utf8Text)
-//                        let bowl = try SwiftSoup.parse(responseContent.debugDescription)
-                        let entries = try bowl.select("div.row>div.col-lg-4")
-                        
-                        return try entries.compactMap {
-                            entry -> AnimeLink? in
-                            let animeTitle = try entry.select("div>h6.card-title>b")
-                            let title = try animeTitle.text()
-                            
-                            if title.isEmpty { return nil }
-                            
-                            let animeLinkPath = try entry.select("div>a")
-                            let url = try animeLinkPath.attr("href")
-                            let animeLinkURLString = url.trimmingCharacters(in: .whitespacesAndNewlines)
-                            let animeArtworkPath = try entry.select("img").attr("src")
-                            
-                            guard let animeUrl = URL(string: animeLinkURLString, relativeTo: endpointUrl),
-                                let coverImage = URL(string: animeArtworkPath, relativeTo: endpointUrl)
-                                else {
-                                    return nil
-                            }
-                            
+                        var encoded = try bowl.select("archivio").attr("records")
+                        encoded = encoded.replacingOccurrences(of: "\n", with: "")
+                        let data_json = encoded.data(using: .utf8)!
```

Comment:
```suggestion
                        let data_json = try encoded.data(using: .utf8).tryUnwrap()
```
Please avoid force casting whenever possible.